cmd/orch-tester/orch_tester.go

```diff
@@ -71,30 +74,26 @@ func main() {
 		log.Fatal("region is required")
 	}
 
-	metricsURL, err := defaultAddr(*metrics, "127.0.0.1", prometheusPort)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	streamTesterURL, err := defaultAddr(*streamTester, "127.0.0.1", streamTesterPort)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	leaderboardURL, err := defaultAddr(*leaderboard, "127.0.0.1", "3001")
-	if err != nil {
-		log.Fatal(err)
-	}
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
 
-	subgraphURL, err := defaultAddr(*subgraph, "127.0.0.1", "8080")
-	if err != nil {
-		log.Fatal(err)
+	if *streamTester == "" {
+		// start streamtester service
+		glog.Info("Using embedded broadcaster")
```

Comment:
typo on broadcaster + log can be the comment
```suggestion
		glog.Info("Starting embedded streamtester service")
```